### PR TITLE
Add swift and python language icons for home page

### PR DESCRIPTION
**Pull request by smharley**
image

### CODE REVIEW

**Comment by croaky** [comment]
Beautiful! Would it be a weird flow to make each logo clickable, and go to the documentation for that language?

**Comment by smharley** [comment]
I think the only downside to that @croaky would be taking people to the documentation rather than putting the CTA to sign up/in right in front of them

**Comment by teoljungberg** [comment]
Have we considered both Python And Swift to be stable?

**Comment by gylaz** [comment]
Swift is stable and we're promoting it in #1065 

@smharley Let's not show the Python icon on the homepage just yet, since it's in beta.

**Comment by croaky** [comment]
> Let's not show the Python icon on the homepage just yet, since it's in beta.

It's been in beta for 6 months. How about putting "Python (beta)" on the home page if there's a beta distinction we want to make? In order to get feedback on the beta, we need people to be aware that it exists.

**Comment by croaky** [comment]
@gylaz thoughts on above?

**Comment by gylaz** [comment]
I think having a beta overlay (or something) for the languages that are in beta would be great.

In regards to Python, it could be that it's been that long, however we haven't gotten much indication about how it's been working. Typically, we'll have a language be in "alpha" -- vet it internally to see how it's working. I think it's been enabled on a few Django projects, but the feedback hasn't been very strong. I will reach out to Python devs again.

**Comment by croaky** [comment]
I don't think we have enough volume internally to depend on pinging people in Slack, Trello, etc. would be good to move each new linter faster to publicly-discoverable, marked as beta in every instance.

**Comment by gylaz** [comment]
We can try that, though we don't have the bandwidth to handle much of the customer support in a timely fashion. Since it's beta, I suppose customers should expect that.

**Comment by smharley** [comment]
I can work on a beta label, I'll update this branch/PR

**Comment by croaky** [comment]
Let's merge this!

**Comment by gylaz** [comment]
I think we should take out Python from this PR. Python has not been working (from the feedback) and the Hound team lacks expertise to debug this.

File: app/views/home/index.haml [hunk 1]
```diff
@@ -38,12 +38,20 @@
             %h5 JavaScript
           %li.language
             .language-logo
+              = image_tag "languages/python.svg"
+            %h5 Python
+          %li.language
+            .language-logo
```
Comment (gylaz) [inline app/views/home/index.haml]: @smharley Can we remove the Python icon from the landing page? (it's in beta and not working very well).

**Comment by smharley** [comment]
Updated screenshot, without Python
image

**Comment by smharley** [comment]
My plan was to leave the Python logo file in, so we can easily add it when the time comes

**Comment by gylaz** [comment]
👍 Good to merge.

**Pull request merged**